project_2_image_captioning_project/code-of-conduct.md

```diff
@@ -0,0 +1,47 @@
+# Code of Conduct
+
+This code of conduct outlines our expectations for participants within the [Student Code-In](https://scodein.tech) community, as well as steps to reporting unacceptable behavior. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community.
```

Comment:
Please remove the StudentCodeIn part. These are general guidelines for all contributors.

Reply:
done